vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_wallet.cc

```diff
@@ -0,0 +1,188 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/json/json_reader.h"
+#include "bat/ledger/internal/ledger_impl.h"
+#include "bat/ledger/internal/uphold/uphold_util.h"
+#include "bat/ledger/internal/uphold/uphold_wallet.h"
+#include "net/http/http_status_code.h"
+
+using std::placeholders::_1;
+using std::placeholders::_2;
+using std::placeholders::_3;
+
+namespace braveledger_uphold {
+
+UpholdWallet::UpholdWallet(bat_ledger::LedgerImpl* ledger, Uphold* uphold) :
+    ledger_(ledger),
+    uphold_(uphold) {
+}
+
+UpholdWallet::~UpholdWallet() {
+}
+
+void UpholdWallet::Generate(
+    std::map<std::string, ledger::ExternalWalletPtr> wallets,
```

Comment:
If the map is really big, we'll be passing a lot of data on the stack. Usually better to use a pointer or const reference in this scenario.

Reply:
map has pointers inside, so I think we should be fine

Reply:
OK.